app/frontend/packs/providers-autocomplete.js

```diff
@@ -0,0 +1,29 @@
+import accessibleAutocomplete from "accessible-autocomplete";
+
+const initProvidersAutocomplete = () => {
+  try {
+    [
+      "#candidate-interface-pick-provider-form-code-field",
+      "#candidate-interface-pick-provider-form-code-field-error"
+    ].forEach(id => {
+      const providersSelect = document.querySelector(id);
+
+      if (!providersSelect) return;
+
+      // Replace "Select a provider" with empty string
+      providersSelect.querySelector("[value='']").innerHTML = "";
+
+      accessibleAutocomplete.enhanceSelectElement({
+        selectElement: providersSelect,
+        showAllValues: true,
+        confirmOnBlur: false
+      });
+
+      console.log(accessibleAutocomplete);
```

Comment:
Begone `console.log`! We have `.eslintrc` configured, which will point these out if you run (or your editor runs) `eslint`